### DISCUSSION TITLE
请问自动生成书签功能如何使用？是出版级的外文PDF书籍。

### DISCUSSION THREAD

**Discussion by AncientTree** [wmjordan/PDFPatcher#94] [问答] [answered]
软件里的一大串选项是在难以理解，操作流程也有点让人摸不着头脑，也没有在您的博客找到任何示例说明，请问如何自动生成书签？用正则表达式不是最简单吗，但我没在软件里找到办法

**Comment by wmjordan** [comment]
把文件发给我看看

**Reply by AncientTree** [reply]
您好，谢谢回复，发到您邮箱了。

**Reply by wmjordan** [reply]
文件好像蛮大的，目前还没收到，等收到之后我看看好不好弄

**Comment by AncientTree** [comment, marked as answer]
| |
PengZhw
|
|
***@***.***
|
在2022年4月17日 ***@***.***> 写道：

把文件发给我看看

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>

从 网易邮箱大师 发来的云附件
|
| |
| |
|
| |
| The Chemistry and Physics of Aerogels Synthesis, Processing, and Properties (Lorenz Ratke, Pavel Gurikov) (z-lib.org).pdf |
| |
| (31.4MB,2022年5月2日 19:13 到期) |
| |
| 下载 |
| |
| |

**Reply by TroyDanielFZ** [reply]
我下载过后，发现这个PDF没有什么特别的，直接常规操作即可。方法如下：
1. 打开文件后，导航到章节头，如P16的“1 Introduction”，把鼠标放到文字上面，可以看到下方状态栏里出来文字“xxx 文本：Introduction”，此时`右击`→`设置自动书签标题格式(Z)`→`1 级标题`，会弹出对话框，第一列出现文字“字体为HelveticaNeueLTStd-MdCn且尺寸为24”。
2. 因为章节标题含有多个字体，此时把鼠标移动到数字1上，添加1级标题。（可`生成书签(K)`看一下结果）
3. 同样的，找一个二级标题，分别在文件和数字上右击添加2级标题，生成书签即可。
4. 修改保存PDF。
@AncientTree

**Reply by AncientTree** [reply]
OHHHHHHHH，awesome!!!!! 我理解错使用逻辑了，感谢你的指导！
